### DISCUSSION TITLE
geoooode

### DISCUSSION THREAD

**Discussion by RapidosElite** [geode-sdk/sdk#527] [General]
can u make geode for ios?

**Comment by RoootTheFox** [comment]
no, it is not possible due to how locked down iOS is.

[edit: i lied](https://github.com/geode-sdk/ios-launcher/blob/main/INSTALL.md)

**Reply by FireMario211** [reply]
this aged badly

**Reply by RoootTheFox** [reply]
oopsies my bad gang

**Comment by RandomGDPlyr** [comment]
my geode is crashing when i open editor

**Reply by TheSillyDoggo** [reply]
womp womp

**Comment by FireMario211** [comment]
geode is on ios now
https://github.com/geode-sdk/ios-launcher/blob/main/INSTALL.md
